src/presentation/http/router/note.test.ts

```diff
@@ -49,49 +50,68 @@ describe('Note API', () => {
   });
 
   describe('GET note/:notePublicId ', () => {
-    test('Returns note with access rights by public id with 200 status when note is publicly available', async () => {
+    test.each([
```

Comment:
Maybe save this test.each as separate util?

something like 
```
test.roles = test.each([{ role: MemberRole.Read,
        isPublic: false,
        isAuthorized: true,
        statusCode: 200 },
        { ... },
        { ... },
        { ... }
]);
```

and use it as
```
test.roles('Returns note with access rights by public id ....', ({ role, isPublic, isAuthorized, statusCode }) => {

})
```

Reply:
I'll think about it, but it's reasonable to do that in another PR

Reply:
deal